packages/address/src/AddressFormatter.ts

```diff
@@ -55,10 +55,7 @@ export default class AddressFormatter {
    */
   async format(address: Address): Promise<string[]> {
     const country = await this.getCountry(address.country);
-    const layout = country.formatting.show || DEFAULT_SHOW_LAYOUT;
-    return layout
-      .split(LINE_DELIMITER)
-      .map(fields => renderLineTemplate(country, fields, address).trim());
+    return format(address, country);
```

Comment:
Can we rename this so there is less confusion with the async `format` above? `buildFormat` maybe?